ui/app/src/model/migrate-client.ts

```diff
@@ -20,7 +20,7 @@ const resource = 'migrate';
 
 export interface MigrateBodyRequest {
   input?: Record<string, string>;
-  grafanaDashboard: string;
+  grafanaDashboard: Record<string, unknown>;
```

Comment:
why do you need a record here ?

Reply:
since I changed the type of `grafanaDashboard` to Record<string, unknown> upstream, I though it would be better to pass it like this to the migrate-client and stringify it just before sending the request, so that we have both args `input` and `grafanaDashboard` behaving the same here. But I dont mind changing it back if you prefer.